submissions/smillims/html-css-popup/#source/scss/style.scss

```diff
@@ -0,0 +1,214 @@
+//nullstyle
+
+@import 'nullstyle.scss';
+
+//font-face
```

Comment:
You can remove this comment as well. It's redundant.
I recommend you using comments in some tricky moments when it's hard to understand something. But in this case, it's clear and easy to read.